### PR TITLE
New Knucklebones crate

### PR DESCRIPTION
**Pull request by lilBlovy**
### About The Pull Request
This PR adds a new crate containing 18 D6 Die to play a game called knuckle bone. Its the bare bones of what you need but I plan on adding more in the future to this crate to make it easier for newer players to learn the game which includes a paper with the rules and a tool to mark the ground. 

### Why It's Good For The Game
 Adds a small game for the community to play. And make it physically possible to play said game because only 11-12 D6 die spawn on every map and people dont want to Ahelp or pray for more die.

### Changelog
🆑 GrimsonGrime
add: new crate containing everything you need to play knucklebone. 
🆑

### CODE REVIEW

**Comment by Mothblocks** [comment]
you want to be using visual studio *code*, not visual studio

**Comment by ExcessiveUseOfCobblestone** [comment]
> you want to be using visual studio _code_, not visual studio

you want to cut ur hair

**Comment by msgerbs** [comment]
Glad you got it fixed. I suggest waiting until after #70210 and then updating this to use the new format though.

**Comment by Mothblocks** [comment]
@ExcessiveUseOfCobblestone  Screen Shot 2022-09-29 at 12 06 04 PM

**Comment by Mothblocks** [comment]
Yeah let's wait on that

**Comment by GrimsonGrime** [comment]
> Yeah let's wait on that

If I may ask what is wrong with it that I could change so that you would want it to be merged in?

**Comment by GrimsonGrime** [comment]
Never mind im stupid like always hahaha

File: code/modules/cargo/packs.dm [hunk 1]
```diff
@@ -2439,6 +2439,31 @@
 					/obj/item/clothing/head/helmet/bluetaghelm)
 	crate_name = "laser tag crate"
 
+/datum/supply_pack/costumes_toys/knucklebone
+	name = "Knucklebone Game Crate"
```
Comment (Mickyan) [inline code/modules/cargo/packs.dm:2443]: Knucklebones with an S at the end

Comment (GrimsonGrime) [reply code/modules/cargo/packs.dm:2443]: The name of the game is Knucklebone dou.

File: code/modules/cargo/packs.dm [hunk 2]
```diff
@@ -2439,6 +2439,31 @@
...
+	desc = "A fun dice game to keep you entertaned for hours and is totaly not ripped off from another game!"
```
Comment (Mickyan) [inline code/modules/cargo/packs.dm:2444]: *entertained

I also feel the need to point out knucklebones is a real ancient game and cult of the lamb did not invent it so nobody is going to get this reference in a few months

Comment (GrimsonGrime) [reply code/modules/cargo/packs.dm:2444]: I am actually extremely glad to hear that!

**Comment by YakumoChen** [comment]
This should be a goody pack, not a crate.

File: code/modules/cargo/packs.dm (hunk 2 above)
Comment (YakumoChen) [reply code/modules/cargo/packs.dm:2444]: Recommend it be "A fun dice game definitely not invented by a cult. Consult your local chaplain regarding approved religious activity."

**Comment by GrimsonGrime** [comment]
> This should be a goody pack, not a crate.
> 
> Or better yet, in the fun vendor.

I plan on adding new things to the crate that will help make the game more engaging. This is kind of just the bare bones to get it out there. That includes a paper with instructions and a hologram projector thing that will allow you to place a grid down to play the game.

**Comment by YakumoChen** [comment]
It should be clear how to play the game in this PR, otherwise it just needs to be labeled "d6 crate"

But again, all of this would be fine in the fun vendor regardless, just stick it all in a dice bag.

**Comment by GrimsonGrime** [comment]
> It should be clear how to play the game in this PR, otherwise it just needs to be labeled "d6 crate"
> 
> But again, all of this would be fine in the fun vendor regardless, just stick it all in a dice bag.
> 
> Also mark it as a draft

I will work on the rules right away. Also how do I mark it?

**Comment by YakumoChen** [comment]
There's a button near the reviews to mark it

File: code/modules/cargo/packs.dm [hunk 3]
```diff
@@ -2439,6 +2439,31 @@
...
+					/obj/item/dice/d6,
+					/obj/item/dice/d6,
+					/obj/item/dice/d6,
+					/obj/item/dice/d6,
+					/obj/item/dice/d6,
+					/obj/item/dice/d6,
+					/obj/item/dice/d6,
+					/obj/item/dice/d6,
+					/obj/item/dice/d6,
+					/obj/item/dice/d6)
```
Comment (Fikou) [inline code/modules/cargo/packs.dm:2463]:
```suggestion
	contains = list(
		/obj/item/dice/d6,
		/obj/item/dice/d6,
		/obj/item/dice/d6,
		/obj/item/dice/d6,
		/obj/item/dice/d6,
		/obj/item/dice/d6,
		/obj/item/dice/d6,
		/obj/item/dice/d6,
		/obj/item/dice/d6,
		/obj/item/dice/d6,
		/obj/item/dice/d6,
		/obj/item/dice/d6,
		/obj/item/dice/d6,
		/obj/item/dice/d6,
		/obj/item/dice/d6,
		/obj/item/dice/d6,
		/obj/item/dice/d6,
		/obj/item/dice/d6,
	)
```

**Comment by JohnFulpWillard** [comment]
Has Been Merged

File: code/modules/cargo/packs.dm (hunk 3 above)
Comment (ExcessiveUseOfCobblestone) [inline code/modules/cargo/packs.dm:2463]:
```suggestion
	contains = list(/obj/item/dice/d6 = 18)
```

File: code/modules/cargo/packs.dm [hunk 4]
```diff
@@ -2439,6 +2439,32 @@
...
+					/obj/item/dice/d6,
+					/obj/item/dice/d6,
+					/obj/item/dice/d6,
+					/obj/item/dice/d6,
+					/obj/item/dice/d6,
+					/obj/item/dice/d6,
+					/obj/item/dice/d6,
+					/obj/item/dice/d6,
+					/obj/item/dice/d6,
+					/obj/item/dice/d6,
```
Comment (JohnFulpWillard) [inline code/modules/cargo/packs.dm:2463]:
```suggestion
	contains = list(/obj/item/dice/d6 = 19,
```

**Comment by JohnFulpWillard** [comment]
Please follow through with @ExcessiveUseOfCobblestone review

**Comment by GrimsonGrime** [comment]
> Please follow through with @ExcessiveUseOfCobblestone review

Done!

File: code/modules/cargo/packs.dm [hunk 5]
```diff
@@ -2439,6 +2439,15 @@
 					/obj/item/clothing/head/helmet/bluetaghelm)
 	crate_name = "laser tag crate"
 
+/datum/supply_pack/costumes_toys/knucklebones
+	name = "Knucklebones Game Crate"
+	desc = "A fun dice game definitely not invented by a cult. Consult your local chaplain regarding approved religious activity."
+	cost = CARGO_CRATE_VALUE * 2
+	contains = list(/obj/item/dice/d6 = 18,
+					/obj/item/paper/guides/knucklebone)
```
Comment (ExcessiveUseOfCobblestone) [inline code/modules/cargo/packs.dm:2447]: should come with like a paint bucket, crayon, or something to make the grid

**Pull request merged**